conf/test_full.config

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
see above

Reply:
Fixed

Reply:
ok so those are the same test files as for the small test right?

In principle it is fine with me however you could for the full_test as it will be running on AWS also choose a different much larger test-dataset. Sth. from a public metabolomics repository that would be to large to run it in the github actions

Reply:
Great point. I was actually thinking about this and we do have a big dataset. But running the workflow with a large dataset will take weeks to finish.